src/main/kotlin/pl/edu/prz/kod/domain/executor/NodeExecutor.kt

```diff
@@ -0,0 +1,19 @@
+package pl.edu.prz.kod.domain.executor
+
+import pl.edu.prz.kod.domain.ExecutionResult
+import pl.edu.prz.kod.domain.Language
+
+class NodeExecutor: AbstractExecutor(Language.NODE) {
+    override fun execute(code: String): ExecutionResult {
+        val escapedCode = code.replace("\"", "\\\"")
```

Comment:
The `\` character has to be espaced as well, because running `console.log("test\" ")` currently fails. The `\` must be escaped before `"`, bacuse otherwise backslahes will be duplicated.
Same issue should also be fixed in `PythonExecutor`. Maybe such escaping of special characters should be extracted to protected function in `AbstractExecutor`, so it could be reused in many executors?